ggml/src/ggml-sycl/common.hpp

```diff
@@ -528,29 +528,26 @@ template <int N, class T> void debug_print_array(const std::string & prefix, con
         ss << array[N - 1];
     }
     ss << "]";
-    GGML_SYCL_DEBUG("%s", ss.str().c_str());
+    return ss.str();
 }
 
-inline void debug_print_tensor(const std::string & prefix, const ggml_tensor * tensor,
-                               const std::string & suffix = "") {
-    if (LIKELY(!g_ggml_sycl_debug)) {
-        return;
-    }
-    GGML_SYCL_DEBUG("%s=", prefix.c_str());
+inline std::string debug_get_tensor_str(const std::string &prefix,
+        const ggml_tensor *tensor, const std::string &suffix = "") {
+    std::stringstream ss;
+    if (LIKELY(!g_ggml_sycl_debug)) { return ss.str(); }
+    ss << prefix.c_str();
```

Comment:
There used to be an `=` sign here printed after the prefix so the in the output log we see something like "dst=\`node_0\`" instead of "dst`node_0`". I see we add the `=` when we use `debug_get_tensor_str` outside of `scope_op_debug_print` so maybe it's best to add the `=` as part of the prefix in `scope_op_debug_print`?

Reply:
Good point and well caught, I added the `=` at end of prefix and removed it from the call to the debug print